src/admin/models/user.ts

```diff
@@ -0,0 +1,25 @@
+export interface User {
```

Comment:
Right, I kept it like this for now just to maintain folder structure, I was thinking about making a shared models file for common models in the future